project1-sps/script.js

```diff
@@ -1,4 +1,39 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+var main = function (playerChoice) {
+  var computerChoice = Math.floor(Math.random() * 3) + 1;
+  if (computerChoice === 1) {
+    computerChoice = "scissors";
+  } else if (computerChoice === 2) {
+    computerChoice = "paper ";
+  } else if (computerChoice === 3) {
+    computerChoice = "stone";
+  }
```

Comment:
this should probably go in its own helper function, the benefits:

1. makes this more modular in case you wanna computer choice more than once
2. abstract complex logic away from your main function